packages/expo-sqlite/ios/ExpoSQLite.podspec

```
@@ -13,7 +13,8 @@ Pod::Spec.new do |s|
   s.author         = package['author']
   s.homepage       = package['homepage']
   s.platforms      = {
-    :ios => '15.1'
+    :ios => '15.1',
+    :osx => "11.0"
```

@Kudo and @alanjhughes will be better persons to review this PR (especially the CRSQLite part), but I just wanted to note that you may also need to replace `ios` platform with `apple` in `expo-module.config.json` so the package is autolinked on Apple platforms other than iOS.

Also, afaik in other packages our minimum macOS version is `10.15`. Is there any reason why you used `11.0`?

```suggestion
    :osx => '10.15',
```

---

Thanks for the initial look, @tsapeta! 

I used `11.0` because that's the target that `npx react-native-macos-init` sets in its `Podfile` by default. Figured I wouldmatch. I don't know of any reason `10.15` wouldn't work here. Haven't tested it yet but I can try it out locally and report back.

---

Ok, let's keep `11.0` then. I didn't know it's been bumped on `react-native-macos` side 😅 We'll probably bump that for other packages too.